src/sentry/tasks/base.py

```diff
@@ -105,10 +107,28 @@ def wrapped(func: Callable[P, R]) -> Task[P, R]:
             wait_for_delivery=wait_for_delivery,
             compression_type=compression_type,
         )(func)
+        if alias or alias_namespace:
```

Comment:
It would be good to expand the doc string for `instrumented_task` to explain how `alias` and `alias_namespace` work.